kpatch/kpatch

```diff
@@ -129,19 +129,19 @@ core_loaded () {
 }
 
 get_module_name () {
-	echo $(readelf -p .gnu.linkonce.this_module $1 | grep '\[.*\]' | awk '{print $3}')
+	readelf -p .gnu.linkonce.this_module "$1" | grep '\[.*\]' | awk '{print $3}'
 }
 
 verify_module_checksum () {
-	modname=$(get_module_name $1)
-	[[ -z $modname ]] && return 1
+	modname="$(get_module_name "$1")"
+	[[ -z "$modname" ]] && return 1
 
-	checksum=$(readelf -p .kpatch.checksum $1 | grep '\[.*\]' | awk '{print $3}')
+	checksum="$(readelf -p .kpatch.checksum "$1" | grep '\[.*\]' | awk '{print $3}')"
 
 	# Fail checksum match only if both exist and diverge
-	if [[ ! -z $checksum ]] && [[ -e "$SYSFS/${modname}/checksum" ]] ; then
-		sysfs_checksum=$(cat $SYSFS/${modname}/checksum)
-		[[ $checksum == $sysfs_checksum ]] || return 1
+	if [[ ! -z "$checksum" ]] && [[ -e "$SYSFS/${modname}/checksum" ]] ; then
+		sysfs_checksum="$(cat "$SYSFS/${modname}/checksum")"
+		[[ "$checksum" == "$sysfs_checksum" ]] || return 1
```

Comment:
Here I think `modname` needs quoting, and `checksum` and `sysfs_checksum` don't need quoting

Reply:
Dito.